FWCore/Utilities/interface/ResolveSymbolicLinks.h

```diff
@@ -0,0 +1,13 @@
+#ifndef FWCore_Utilities_ResolveSymbolicLinks_h
```

Comment:
Please change the name of the file to start with a lower case since this isn't a class header.